Source/PoolingFunctions/arm_avgpool_s16.c

```diff
@@ -91,200 +91,228 @@ arm_cmsis_nn_status arm_avgpool_s16(const cmsis_nn_context *ctx,
     const int32_t act_min = pool_params->activation.min;
     const int32_t act_max = pool_params->activation.max;
     const int32_t ch_src = input_dims->c;
+    const int32_t batch_input = input_x * input_y * ch_src;
+    int32_t batch_cnt = input_dims->n;
```

Comment:
Consider a parameter validity check here(in other places as well) so any previously unset case is not silently ignored..

Reply:
Done